### PR TITLE
Added support for Beta channel deployment

### PR DESCRIPTION
**Pull request by FrenchBen**
### Proposed changes

Add beta channels to Docker for AWS and Docker for Azure

### Unreleased project version (optional)

Beta links will not work until the 1.13.1-rc has been released.

### CODE REVIEW

File: docker-for-aws/faqs.md [hunk 1]
```diff
@@ -4,11 +4,29 @@ keywords: aws faqs
 title: Docker for AWS Frequently asked questions (FAQ)
 ---
 
-## Can I use my own AMI?
+### Stable and beta channels
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: Please make sure you start with level-2 headings. If you skip a step, Jekyll can get very confused.

File: docker-for-aws/faqs.md [hunk 2]
```diff
@@ -4,11 +4,29 @@ keywords: aws faqs
...
+  feedback as the deployment evolve. Releases are typically more frequent than for
+  stable, often one or more per month. Usage statistics and crash reports are sent
+  by default. You do not have the option to disable this on the beta channel.
+
+### Can I use my own AMI?
 
 No, at this time we only support the default Docker for AWS AMI.
 
-## How to use Docker for AWS with an AWS account in an EC2-Classic region.
+### How to use Docker for AWS with an AWS account in an EC2-Classic region.
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: How can I use Docker for AWS with an AWS account in an EC2-Classic region?

File: docker-for-aws/faqs.md [hunk 3]
```diff
@@ -39,27 +57,27 @@ There are a few work arounds that you can try to get Docker for AWS up and runni
 - https://aws.amazon.com/blogs/aws/amazon-ec2-update-virtual-private-clouds-for-everyone/
 
 
-## Can I use my existing VPC?
+### Can I use my existing VPC?
 
 Not at this time, but it is on our roadmap for future releases.
 
-## Which AWS regions will this work with.
+### Which AWS regions will this work with.
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: Make it a question

File: docker-for-aws/faqs.md [hunk 4]
```diff
@@ -39,27 +57,27 @@ There are a few work arounds that you can try to get Docker for AWS up and runni
...
 
 Docker for AWS should work with all regions except for AWS China, which is a little different than the other regions.
 
-## How many Availability Zones does Docker for AWS use?
+### How many Availability Zones does Docker for AWS use?
 
 All of Amazons regions have at least 2 AZ's, and some have more. To make sure Docker for AWS works in all regions, only 2 AZ's are used even if more are available.
 
-## What do I do if I get "KeyPair error" on AWS?
+### What do I do if I get "KeyPair error" on AWS?
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: What do I do if I get `KeyPair error` on AWS?

File: docker-for-aws/faqs.md [hunk 5]
```diff
@@ -39,27 +57,27 @@ There are a few work arounds that you can try to get Docker for AWS up and runni
...
 As part of the prerequisites, you need to have an SSH key uploaded to the AWS region you are trying to deploy to.
 For more information about adding an SSH key pair to your account, please refer to the [Amazon EC2 Key Pairs docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html)
 
-## Where are my container logs?
+### Where are my container logs?
 
 All container logs are aggregated within [AWS CloudWatch](https://aws.amazon.com/cloudwatch/).
 
-## I have a problem/bug where do I report it?
+### I have a problem/bug where do I report it?
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: Where do I report problems or bugs?

File: docker-for-aws/faqs.md [hunk 6]
```diff
@@ -77,11 +95,11 @@ Please provide this session ID to the maintainer debugging your issue.
 
 _Please note that your output will be slightly different from the above, depending on your swarm configuration_
 
-## Analytics
+### Analytics
 
 Docker for AWS sends anonymized minimal analytics to Docker (heartbeat). These analytics are used to monitor adoption and are critical to improve Docker for AWS.
 
-## How to run administrative commands?
+### How to run administrative commands?
```
Comment (mstanleyjones) [inline docker-for-aws/faqs.md]: How do I run administrative  commands?

File: docker-for-azure/faqs.md [hunk 7]
```diff
@@ -69,3 +69,34 @@ $ sudo ping 10.0.0.4
 ```
 
 Note that access to Docker for Azure and Azure happens through a shell container that itself runs on Docker.
+
+
+### What are the different Azure Regions?
+All regions can be found here: https://azure.microsoft.com/en-us/regions/
+An exerpt of the above regions to use when you create your service principal are:
+```
```
Comment (mstanleyjones) [inline docker-for-azure/faqs.md]: You need a newline after line 76.

Do we really want to list out all these regions? What if they change? Isn't it sufficient to have the link? (btw you need to linkify the link, that doesn't happen automatically) and maybe an example specifying a region?

Comment (FrenchBen) [inline docker-for-azure/faqs.md]: The link shows the different regions, but doesn't actually provide the region code - If new regions are added, we can update it; this should be a good list for most.

**Pull request merged**